fastestimator/estimator.py

```diff
@@ -53,31 +54,26 @@ class Estimator:
     pipeline: Pipeline
     traces: List[Trace]
```

Comment:
This is now:

traces: List[Union[Trace, Scheduler[Trace]]]

Reply:
done